bin/cron/build_contact_rollups

```diff
@@ -3,11 +3,43 @@
 require_relative('../../dashboard/config/environment')
 require 'cdo/contact_rollups'
 require 'cdo/contact_rollups_validation'
+require 'cdo/pardot'
+require 'cdo/chat_client'
 require 'cdo/only_one'
 
+CHAT_CHANNEL = "cron daily".freeze
```

Comment:
nit: Existing chat rooms seem to follow the convention of hyphens (not spaces) between words.